src/Interceptor/SetRequestTimeout.php

```diff
@@ -10,15 +10,18 @@ public function __construct(
         float $tcpConnectTimeout = 10,
         float $tlsHandshakeTimeout = 10,
         float $transferTimeout = 10,
+        float $inactivityTimeout = 10,
```

Comment:
Should this default to `null` to prevent modifying the inactivity timeout if there's already a custom timeout present?

Reply:
You mean for BC?

Reply:
Yes

Reply:
Changed!